cli/code_scanner.py

```diff
@@ -391,6 +423,48 @@ def exclude_irrelevant_scan_results(document_detections_list: List[DocumentDetec
     return relevant_document_detections_list
 
 
+def parse_pre_receive_input() -> str:
+    pre_receive_input = sys.stdin.read().strip()
```

Comment:
Add an example for the pre-receive input, so it will be easier to understand

```
old_value new_value refname
-----------------------------------------------
0000000000000000000000000000000000000000 9cf90954ef26e7c58284f8ebf7dcd0fcf711152a refs/heads/main
0000000000000000000000000000000000000000 973a96d3e925b65941f7c47fa16129f1577d499f refs/heads/feature-branch
```